FantasmoSDK/Classes/Extensions/ARFrame+Extension.swift

```diff
@@ -8,8 +8,96 @@
 import ARKit
 
 extension ARFrame {
-    /// Calculate the FMPose difference of self with respect to the given frame
-    func poseWithRespectTo(_ frame: ARFrame) -> FMPose {
-        return FMPose.diffPose(self.camera.transform, withRespectTo: frame.camera.transform)
+    
+    /// Transform of device in coordinate system of camera.
+    /// Coordinate system of a device is turned by pi/2 about z-axis relative to coordinate system of camera. For more details see comment to
+    /// `ARFrame.deviceTransform`.
+    private static let transformOfDeviceCS_inCameraCS =
```

Comment:
These variable names do not follow Swift conventions. Let's stick to Swift style, as with `transformOfWorldCSInOpenCvCS` below.

Reply:
Yes, I agree, some useful notes can be found at https://docs.microsoft.com/en-us/previous-versions/dotnet/netframework-1.1/141e06ef(v=vs.71)?redirectedfrom=MSDN